_targets.R

```diff
@@ -223,8 +225,12 @@ tar_target(rs_school_fastest, {
   rs
 }),
 
+tar_target(done_school_fastest, {
+  exists(rs_school_fastest) #Hack for scheduling
```

Comment:
This will still load into memory the whole object. Worth opening an issue in the targets issue tracker IMO.